packages/web-components/src/components/va-file-input/va-file-input.tsx

```diff
@@ -33,6 +33,7 @@ export class VaFileInput {
   private fileInputRef!: HTMLInputElement;
   private uploadStatus: 'idle' | 'success' = 'idle';
   private fileType?: string;
+  private fileInputString: string ='choose from folder';
```

Comment:
Will this string value ever change? Or would the variable be better defined as a constant?